client/connect-account-page-experiment/index.js

```diff
@@ -0,0 +1,25 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Experiment } from '@woocommerce/explat';
+
+/**
+ * Internal dependencies
+ */
+import ConnectAccountPage from 'connect-account-page';
+import ConnectAccountPageVariant from 'connect-account-page-variant';
+
+const ConnectAccountPageExperiment = () => {
+	return (
+		<Experiment
```

Comment:
On the ExPlat side of things this looks straightforward 👍 
- It might be helpful to preload the experiment assignment one "step" before this if possible to reduce the chance of a loading state.
- Have you tested the loadingExperience state? Is it ok to return `null` here?

Other than that LGTM

Reply:
Thanks @jessie-ross !

> Have you tested the loadingExperience state? Is it ok to return null here?

The delay in loading has seemed imperceivable in testing so far. Thinking about this more, it would be more ideal to have a loading indicator, especially for occasions when loading does take longer.

> It might be helpful to preload the experiment assignment one "step" before this if possible to reduce the chance of a loading state.

Good idea!

Would we be able to implement these changes in the next release of WC Pay (3 weeks time), but still go ahead with the experiment in the current release?

Reply:
> Would we be able to implement these changes in the next release of WC Pay (3 weeks time), but still go ahead with the experiment in the current release?

If you are fine with the current behaviour (it isn't bad, just not ideal) I would say go for it, otherwise we would be waiting 3 more weeks to run the experiment.

> The delay in loading has seemed imperceivable in testing so far. Thinking about this more, it would be more ideal to have a loading indicator, especially for occasions when loading does take longer.

It might be worth testing with emulated slow internet connections to be sure it is ok.

Reply:
Oh and if it is already released, users are already experiencing the loading delay

Reply:
Yeah this ships in the version released today 😅 

I'm thinking we should improve it for the next release, and then it will also serve as a good example for future experiments to follow.

Reply:
Well the experiment should be running this release and I would expect it to be removed next release.